packages/collector/test/tracing/database/db2/test.js

```diff
@@ -880,10 +873,11 @@ mochaSuiteFn('tracing/db2', function () {
         .then(() =>
           testUtils.retry(() =>
             verifySpans(agentControls, controls, {
+              // Spans:
               // 11 queries splitted from the file
               // 4 fs operations on top (ours + from db2 internally fs-extra)
               // https://github.com/ibmdb/node-ibm_db/blob/fb25937524d74d25917e9aa67fb4737971317986/lib/odbc.js#L916
-              numberOfSpans: oTelIntegrationIsEnabled ? 15 : 11,
+              // If the Otel integration is disabled, we expect 11 spans.
```

Comment:
_question:_ If we no longer test for the exact number of spans, can we delete this comment entirely? I don't think it serves a purpose anymore.